src/Message/PaymentRedirectResponse.php

```diff
@@ -5,10 +5,7 @@
 use Omnipay\Common\Message\AbstractResponse;
 use Omnipay\Common\Message\RedirectResponseInterface;
 
-/**
- * PayZen Purchase Response
- */
-class PurchaseResponse extends AbstractResponse implements RedirectResponseInterface
+class PaymentRedirectResponse extends AbstractResponse implements RedirectResponseInterface
```

Comment:
`PayzenResponse` ?

Reply:
It's, for me, less clear. It seems like the payzen response while this is instead the redirect to payzen page.

Reply:
`PayzenRedirectResponse` then.

Reply:
Do you like `redirectToGatewayResponse` ? @everneret

Reply:
Let's settle for this :wink: